frontend/src/lib/components/Lettermark/Lettermark.tsx

```diff
@@ -3,30 +3,41 @@ import React from 'react'
 import './Lettermark.scss'
 
 export enum LettermarkColor {
-    Default = 'default',
-    Gray = 'gray',
+    Default = 1,
+    Gray = 8,
 }
 
 export interface LettermarkProps {
     /** Name or value the lettermark should represent. */
     name?: string | number | null
+    // If given, will choose a color based on the index
```

Comment:
Tip: Use JSDoc syntax `/** Lorem ipsum */` for these comments, that way they get helpfully shown in tooltips by IDEs